controllers/app.controller.js

```diff
@@ -9,3 +9,14 @@ exports.getTopics = (req, res) => {
       console.log(err);
     });
 };
+
+exports.getArticle = (req, res) => {
+  const { article_id } = req.params;
+  selectArticle(article_id)
+    .then((result) => {
+      res.status(200).send({ result });
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
we need to be adding some proper error handling at this stage - see review comment